### PR TITLE
add langchain example

### PR DESCRIPTION
**Pull request by zeroxbt**

### CODE REVIEW

File: examples/langchain/dkg-demo.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import "dotenv/config";
+import DKG from "dkg.js";
+import fs from "fs";
+
+const yewmakerol = JSON.parse(fs.readFileSync("./yewmakerol.json"));
```
Comment (branarakic) [inline examples/langchain/dkg-demo.js:8]: let's add some inline comments to explain what is happening here (why yewmakerol.json)
would also be good to point to a demo video from the comments

File: examples/langchain/generate-tsv.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import fs from "fs";
+
```
Comment (branarakic) [inline examples/langchain/generate-tsv.js:4]: Also would add some explanatory comments here

File: examples/langchain/search.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import json
+from dotenv import load_dotenv
+
+import openai
+from langchain.embeddings import HuggingFaceEmbeddings
+from langchain.vectorstores import Milvus
+
+load_dotenv()
+
+
```
Comment (branarakic) [inline examples/langchain/search.py:11]: Some basic comments would provide an easier way to navigate and understand the code a well

File: examples/langchain/README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
+# ChatDKG Project README
+
+This example walks you through the process of creating a knowledge asset on the Origintrail decentralized knowledge graph and searching the knowledge asset using the power of Langchain and OpenAI.
```
Comment (branarakic) [inline examples/langchain/README.md]: Let's provide some more context in the readme: what are KAs, reference Milvus, explain in a few steps what the algo looks like, include links where appropriate, including the video where we showed this demo (ideally at the point of showing it)

**Review by branarakic — CHANGES_REQUESTED** [review]

File: examples/langchain/README.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+- Python 3.10 or higher.
+- An account on [Milvus](https://cloud.zilliz.com/orgs).
+
+## Installation
+
+Clone the repository:
+
+```bash
+git clone https://github.com/Origintrail/ChatDKG
+cd ChatDKG
```
Comment (NZT48) [inline examples/langchain/README.md:25]:
```suggestion
cd ChatDKG/examples/langchain
```

File: examples/langchain/dkg-demo.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+const dkg = new DKG({
+  endpoint: process.env.OT_NODE_HOSTNAME,
+  blockchain: {
+    name: "otp::testnet",
```
Comment (NZT48) [inline examples/langchain/dkg-demo.js:15]: I would put into readme link to our discord link and say that they can get tokens from our faucet on discord

File: examples/langchain/README.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
+# ChatDKG Project README
+
+This example walks you through the process of creating a knowledge asset on the Origintrail decentralized knowledge graph and searching the knowledge asset using the power of Langchain and OpenAI.
+
+## Pre-requisites
+
+- NodeJS v16 or higher.
+- Python 3.10 or higher.
+- An account on [Milvus](https://cloud.zilliz.com/orgs).
+
```
Comment (NZT48) [inline examples/langchain/README.md:18]: Should we add also OpenAi API key as prerequisite

**Review by NZT48 — CHANGES_REQUESTED** [review]

File: examples/langchain/README.md (hunk 7 above)
Comment (branarakic) [reply examples/langchain/README.md:18]: It's optional, but should be indicated

**Pull request merged**